### DISCUSSION TITLE
Export backrest repo

### DISCUSSION THREAD

**Discussion by PFedya8** [garethgeorge/backrest#188] [Q&A] [answered]
Hello, I set up backup for some folders in my vm (I use backrest and s3 to store snapshots) and I have next question, if something happens with my server will I be able to access files from another virtual machine? And how can I do this?
Thanks in advance!

**Comment by garethgeorge** [comment, marked as answer]
Save a copy of your backrest config in a safe location, you’ll be able to add that configuration to a new install in a new virtual machine and will be able to import your snapshots and restore from there. I keep my configs as a files in my password manager.

**Reply by PFedya8** [reply]
Thanks!
